platform-hub-api/app/services/kubernetes/static_token_service.rb

```diff
@@ -0,0 +1,82 @@
+module Kubernetes
+  module StaticTokenService
+    extend self
+
+    # Static tokens are stored in HashRecord as:
+    # data: [
+    #   {"token" => "token1", user" => "user1", "uid" => "uid1", "groups" => ["group1","group2"]}
+    #   {"token" => "token2", user" => "user2", "uid" => "uid2", "groups" => []}
+    #   ...
+    # ]
+    # Note: All tokens are encrypted!
+
+    def create_or_update(cluster, kind, user_name, groups = [])
+      static_tokens = get_static_tokens(cluster, kind)
+
+      record = static_tokens.data.find do |t|
+        t['user'] == user_name.to_s
+      end
+
+      if record.nil?
+        token = Kubernetes::TokenService.generate_secure_random
+        record = Hashie::Mash.new(
```

Comment:
Does this need to be a `Hashie::Mash` instance? (doesn't look like it's used later on in the method, so the convenience provided by Mash is probably not worth it?)